torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -167,6 +170,7 @@ void PyTorchModelLoader::loadAdd(const torch::jit::Node *ptNode) {
   const auto scalarHandle = getGlowConstantHandle<int32_t>(inputs[2]);
   assert(scalarHandle.size() == 1);
   assert(scalarHandle.raw(0) == 1);
+  (void)scalarHandle;
```

Comment:
Wonder what is this for?

Reply:
scalarHandle is only used in `assert()` so it will look as though it's unused when compiled in Release so this makes it so the compiler doesn't complain about that.